eth/v1alpha1_gateway/README.md

```diff
@@ -1,4 +0,0 @@
-# gRPC Gateway
-
-This package is contains generated files for applications that wish to use eth/v1alpha as a 
-[gRPC gateway](https://github.com/grpc-ecosystem/grpc-gateway).
```

Comment:
Do you want to just delete this file? seems weird to clear out all of the contents.

Reply:
Replaced with a README in v1/ and v1alpha1/